src/OMSimulatorLib/TLM/SystemTLM.cpp

```diff
@@ -506,7 +506,7 @@ oms_status_enu_t oms3::SystemTLM::updateInitialValues(const oms3::ComRef cref)
 oms_status_enu_t oms3::SystemTLM::initializeSubSystem(oms3::ComRef cref)
 {
   oms_status_enu_t status = getSubSystem(cref)->initialize();
-  if(status = oms_status_ok)
+  if(status == oms_status_ok)
```

Comment:
@robbr48 To avoid such typos, I usually write `if (oms_status_ok == status)`. That way, the typo would cause a compilation error.

Reply:
Aha, so that's the reason. I have been wondering why you do it that way :) I will have a look at the failing tests mentioned below.